include/zephyr/ll_sched/ll_scheduler.h

```diff
@@ -0,0 +1,303 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Bartosz Kokoszko <bartoszx.kokoszko@linux.intel.com>
+ */
+
+#ifndef __LL_SCHEDULER_H__
+#define __LL_SCHEDULER_H__
+
+#include <zephyr/kernel.h>
+
+#include <assert.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/** \brief Type of scheduler, comes from topology. */
```

Comment:
the file has a mix of `\brief`, `@brief`, and implicit briefs. I would suggest to use the same approach for the entire file (I think implicit works nicely).

Reply:
Will use @brief moving forward as that appears to be most common in our source.